CodeGen/Generators/UnitsNetGen/StaticQuantityGenerator.cs

```diff
@@ -22,6 +22,8 @@ public override string Generate()
 using UnitsNet.InternalHelpers;
 using UnitsNet.Units;
 
+#nullable enable
```

Comment:
Shouldn't we set this on the project instead?
https://docs.microsoft.com/en-us/dotnet/csharp/tutorials/nullable-reference-types

Or are there so many issues to fix that we will have to introduced this in bits and pieces?

Reply:
Other than that, this looks pretty good to me and ready to merge.

Reply:
It is set on the project. However without this in the generated files, I receive a compiler error:

```
Error CS8669 The annotation for nullable reference types should only be used in code within a '#nullable' annotations context. Auto-generated code requires an explicit '#nullable' directive in source.
```

I'm guessing the .g. annotation tells the compiler it's generated?

Reply:
Aha, yes it probably does. Today I learned.